src/Host/Client/Impl/Definitions/SendBlobResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.R.Host.Client {
+    public class SendBlobResult : BlobResult {
```

Comment:
It feels like this, and `GetBlobResult`, do not really offer anything useful to the API client - they won't care about `Kind`, because the methods that are exposed to them return specific subtypes anyway. They'll just want the ID for send, and the data for get. We should make this private to `RHost`, and expose the results directly on private `IRBlobHelper` methods.